cross/wallabag/Makefile

```diff
@@ -0,0 +1,23 @@
+PKG_NAME = wallabag
+PKG_VERS = 1.9.1-b
+PKG_EXT = zip
+PKG_DIST_NAME = $(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/wallabag/wallabag/archive
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS =
+
+HOMEPAGE = https://www.wallabag.org/
+COMMENT  = Wallabag is a self hostable application allowing you to not miss any content anymore. Click, save, read it when you can. It extracts content so that you can read it when you have time.
```

Comment:
...allowing you to save an offline copy of your favorite articles. 

sounds a bit better, don't you think ?